specification/eventgrid/data-plane/Microsoft.ResourceNotifications/stable/2018-01-01/common.json

```diff
@@ -0,0 +1,105 @@
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "id of the resource for which the event is being emitted",
+          "type": "string"
+        },
+        "name": {
+          "description": "name of the resource for which the event is being emitted",
+          "type": "string"
+        },
+        "type": {
+          "description": "the type of the resource for which the event is being emitted",
+          "type": "string"
+        },
+        "location": {
+          "description": "the location of the resource for which the event is being emitted",
+          "type": "string"
+        },
+        "tags": {
+          "description": "the tags on the resource for which the event is being emitted",
+          "type": "string"
+        },
+        "properties": {
+          "description": "properties in the payload of the resource for which the event is being emitted",
+          "type": "object",
+          "additionalProperties": {
+            "description": "properties in the payload of the resource for which the event is being emitted"
+          }
+        }
+      }
+    },
+    "ResourceNotificationsResourceWithoutPayloadDetails": {
+      "description": "Describes the schema of the common properties across all ARN system topic deleted events",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "id of the resource which is being deleted",
+          "type": "string"
+        },
+        "name": {
+          "description": "name of the resource being deleted",
+          "type": "string"
+        },
+        "type": {
+          "description": "the type of the resource being deleted",
+          "type": "string"
+        }
+      }
+    },
+    "ResourceNotificationsOperationalDetails": {
+      "description": "details of operational info",
+      "type": "object",
+      "properties": {
+        "resourceEventTime": {
+          "description": "Date and Time when resource was updated",
+          "type": "string",
+          "format": "date-time"
+        }
+      }
+    },
+    "ResourceNotificationsResourceUpdatedDetails": {
+      "description": "Describes the schema of the common properties across all ARN system topic events",
+      "type": "object",
+      "properties": {
+        "resourceInfo": {
+          "description": "resourceInfo details for update event",
+          "$ref": "#/definitions/ResourceNotificationsResourceWithPayloadDetails"
+        },
+        "operationalInfo": {
+          "description": "details about operational info",
+          "$ref": "#/definitions/ResourceNotificationsOperationalDetails"
+        },
+        "apiVersion": {
+          "description": "api version of the resource properties bag",
+          "type": "string"
+        }
+      }
+    },
+    "ResourceNotificationsResourceDeletedDetails": {
```

Comment:
Is this still being used? If not, we should delete it for now.

Reply:
done!